FinniversKit/Sources/Recycling/GridViews/NeighborhoodProfile/NeighborhoodProfileView.swift

```diff
@@ -20,7 +20,30 @@ public final class NeighborhoodProfileView: UIView {
 
     // MARK: - Private properties
 
-    private var viewModel = NeighborhoodProfileViewModel(title: "", readMoreLink: nil, cards: [])
+    private var viewModel = NeighborhoodProfileViewModel(title: "", readMoreLink: nil, cards: [], banner: nil)
+
+    private lazy var bannerView: NeighborhoodProfileBannerView = {
+        let view = NeighborhoodProfileBannerView(withAutoLayout: true)
+        view.delegate = self
+        return view
+    }()
+
+    private func centeredContainerView(with views: [UIView]) -> UIStackView {
+        let stack = UIStackView(withAutoLayout: true)
+        stack.axis = .horizontal
+        stack.isLayoutMarginsRelativeArrangement = true
+        stack.directionalLayoutMargins = NSDirectionalEdgeInsets(vertical: 0, horizontal: .spacingM)
+        stack.addArrangedSubviews(views)
+        return stack
+    }
+
+    private lazy var containerStackView: UIStackView = {
+        let stack = UIStackView(withAutoLayout: true)
+        stack.axis = .vertical
+        stack.spacing = .spacingS
+        stack.alignment = .fill
+        return stack
+    }()
```

Comment:
We have a shorthand for this, which can be used instead:

```swift
private lazy var containerStackView = UIStackView(axis: .vertical, spacing: .spacingS, withAutoLayout: true)
```

UIStackView defaults both `alignment` and `distribution` to `.fill`, so that's not needed :)